flake.lock

```diff
@@ -0,0 +1,132 @@
+{
+  "nodes": {
+    "fenix": {
+      "inputs": {
+        "nixpkgs": [
+          "nixpkgs"
+        ],
+        "rust-analyzer-src": "rust-analyzer-src"
```

Comment:
Correct me if I'm wrong, we need the language server, because this env will not just be used for building Iroha, but also for developing it, right?

Reply:
Yes, this probably comes from `devShell`
iroha derivation pulls in only necessary parts of toolchain